core/src/main/java/nl/nn/adapterframework/validation/IXSD.java

```diff
@@ -41,7 +42,7 @@ default boolean hasDependency(Set<IXSD> xsds) {
 		return false;
 	}
 
-	Set<IXSD> getXsdsRecursive(boolean supportRedefine) throws ConfigurationException;
+	void getXsdsRecursive(Map<String,IXSD> xsds, boolean supportRedefine) throws ConfigurationException;
```

Comment:
Waarom is dit eigenlijk als public method gedefinieerd in the interface?
Er is maar 1 implementatie van, en die wordt alleen intern gebruikt in `XSD.java`.